.github/workflows/create-quantic-package.yml

```diff
@@ -0,0 +1,56 @@
+name: Create Quantic Package Workflow
+
+env:
+  SFDX_AUTH_JWT_INSTANCE_URL: https://login.salesforce.com
+  SFDX_AUTH_JWT_USERNAME: sfdc.integration.devv2.hub@coveo.com
+  SFDX_AUTH_JWT_KEY: ${{ secrets.SFDX_AUTH_JWT_KEY_DEV }}
+  SFDX_AUTH_JWT_CLIENT_ID: ${{ secrets.SFDX_ATH_JWT_CLIENT_ID }}
+  PACKAGE_ID: 0Ho6g000000k9g8CAA
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '30 17 * * *'
```

Comment:
This is 17:30 UTC which means 13h30 for us. I guess we need more a nightly kind of build, so I would push it to 18h instead. The cron job would change to `0 22 * * *`.